crates/sargon/src/signing/collector/signatures_collector.rs

```diff
@@ -45,8 +45,7 @@ impl<S: Signable> SignaturesCollector<S> {
     }
 
     pub async fn collect_signatures(self) -> Result<SignaturesOutcome<S::ID>> {
-        self
-            .sign_with_factors() // in decreasing "friction order"
+        self.sign_with_factors() // in decreasing "friction order"
```

Comment:
When I did this change, swift format changed it back to two lines. I don't understand why.

Reply:
did `swiftformat` do that? Swift? Not Cargo fmt?

Reply:
Meant fmt, sorry, force of recent habit

Reply:
ok strange, we can look into it, that `clippy` and `fmt` should both use same rules...

Reply:
Although earlier this method looked like this:

```
let _ = self
            .sign_with_factors()
```

Reply:
the `let _ = ` is not really needed I guess